src/coreclr/src/vm/dwbucketmanager.hpp

```diff
@@ -954,7 +954,7 @@ bool BaseBucketParamsManager::GetFileVersionInfoForModule(Module* pModule, USHOR
     CONTRACTL
     {
         NOTHROW;
-        GC_NOTRIGGER;
+        GC_TRIGGERS;
```

Comment:
possibly leave this as `GC_NOTRIGGER`?

Reply:
`GC_TRIGGERS` vs. `GC_NOTRIGGER` does not really matter in combination with `MODE_PREEMPTIVE`.